language/stdlib/README.md

```diff
@@ -0,0 +1,66 @@
+---
+id: libra-framework
+title: Libra Framework
+custom_edit_url: https://github.com/libra/libra/edit/master/language/stdlib/README.md
+---
+
+## The Libra Framework
+
+The Libra Framework defines the standard actions that can be performed on-chain
+both by the Libra VM---through the various prologue/epilogue functions---and by
+users of the blockchain---through the allowed set of transactions. This
+directory contains different directories that hold the source Move
+modules and transaction scripts, along with a framework for generation of
+documentation, ABIs, and error information from the Move source
+files. See the [Layout](#layout) section for a more detailed overview of the structure.
+
+## Documentation
+
+Each of the main components of the Libra Framework and contributing guidelines are documented separately. Particularly:
+* Documentation for the set of allowed transaction script can be found in [transaction_scripts/doc/transaction_script_documentation.md](transaction_scripts/doc/transaction_script_documentation.md).
+* The overview documentation for the Move modules can be found in [modules/doc/overview.md](modules/doc/overview.md).
+* Contributing guidelines and basic coding standards for the Libra Framework can be found in [CONTRIBUTING.mod](CONTRIBUTING.md).
+
+## Compilation and Generation
+
+Recompilation of the Libra Framework and the regeneration of the documents,
+ABIs, and error information can be performed by running `cargo run` from this
+directory. There are a number of different options available and these are
+explained in the help for this command by running `cargo run -- --help` in this
+directory. Compilation and generation will be much faster if run in release
+mode (`cargo run --release`).
+
+## Layout
+The overall structure of the Libra Framework is as follows:
+
+```
+├── compiled                                # Generated files and public rust interface to the Libra Framework
+│   ├── error_descriptions                  # Generated error descriptions for use by the Move Explain tool
+│   ├── src                                 # External Rust interface/library to use the Libra Framework
+│   ├── stdlib                              # The compiled Move bytecode of the Libra Framework source modules
+│   │   ├── 000_Debug.mv
+│   │   ...
+│   │   └── 039_SharedEd25519PublicKey.mv
+│   └── transaction_scripts                 # Generated ABIs for each transaction script in the allowlist
+│       ├── abi
+│       │   ├── add_currency_to_account.abi
+│       │   ...
+│       │   └── update_minting_ability.abi
+│       ├── add_currency_to_account.mv
+│       ...
+│       └── update_minting_ability.mv
+├── modules                                 # Libra Framework source modules
+│   ├── AccountFreezing.move
```

Comment:
Ditto. This one is especially confusing because the "doc" directory occurs alphabetically in the middle of the example files. "*.move" and "doc/*.md" would be much better